chia/util/streamable.py

```diff
@@ -295,11 +351,21 @@ def function_to_parse_one_item(cls: Type[cls.__name__], f_type: Type):  # type:
     def parse(cls: Type[cls.__name__], f: BinaryIO) -> cls.__name__:  # type: ignore
         # Create the object without calling __init__() to avoid unnecessary post-init checks in strictdataclass
         obj: Streamable = object.__new__(cls)
-        fields: Iterator[str] = iter(getattr(cls, "__annotations__", {}))
+        try:
+            bare_fields = cls._chia_streamable.fields
+        except AttributeError:
+            bare_fields = []
+        fields: Iterator[str] = iter(field.name for field in bare_fields)
         values: Iterator = (parse_f(f) for parse_f in PARSE_FUNCTIONS_FOR_STREAMABLE_CLASS[cls])
         for field, value in zip(fields, values):
             object.__setattr__(obj, field, value)
 
+        # TODO: should we handle inheritance here?  Or should the post init super() for
+        #       itself?
+        post_init = getattr(cls, "__post_init__", None)
```

Comment:
```suggestion
        post_init = getattr(cls, "__post_init__", lambda *args, **kwargs: 0)
```

then you can just call it no matter what.

Reply:
No biggie either way really.